harmonica/forward/point_mass.py

```diff
@@ -7,28 +7,48 @@
 from ..constants import GRAVITATIONAL_CONST
 
 
-def point_mass_gravity(coordinates, points, masses, field, dtype="float64"):
+def point_mass_gravity(
+    coordinates, points, masses, field, coordinate_system="cartesian", dtype="float64"
+):
     """
     Compute gravitational fields of a point mass on spherical coordinates.
```

Comment:
It would be good to elaborate a bit more on the docstring (citation, definitions, equations [?]).

Reply:
Yes, absolutely! I tend to forget to do it because I know where I'm getting the equations from, but I really appreciate finding math on the docstrings when I try to use a new library or function.

Reply:
I added some math and references. I think we must make clearer that the third coordinate in Cartesian system is pointing downwards.

Reply:
The warning mentioned above should be enough for making it very clear.